stream-product/product-ingestion-saga/src/main/java/com/backbase/stream/product/BusinessFunctionGroupMapper.java

```diff
@@ -1,12 +1,18 @@
 package com.backbase.stream.product;
 
 import com.backbase.dbs.accesscontrol.api.service.v2.model.FunctionGroupItem;
+import com.backbase.dbs.accesscontrol.api.service.v2.model.Permission;
+import com.backbase.stream.legalentity.model.BusinessFunction;
 import com.backbase.stream.legalentity.model.BusinessFunctionGroup;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
 
 @Mapper
 public interface BusinessFunctionGroupMapper {
 
+  @Mapping(source = "permissions", target = "functions")
   BusinessFunctionGroup map(FunctionGroupItem functionGroupItem);
 
+  @Mapping(source = "assignedPrivileges", target = "privileges")
+  BusinessFunction mapBusinessFunction(Permission permission);
```

Comment:
I don't see any code where this newly added method is being used, check you re-check

Reply:
@tarun-bb This is the fix which resolves the custom job role issue, the mapping for permissions was not there and by adding that mapping, the issue gets fixed.